задани1/StructVector3DLibrary/Vector3D.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StructVector3DLibrary
+{
+    public struct Vector3D
+    {
+        public double X { get; set; }
+        public double Y { get; set; }
+        public double Z { get; set; }
+
+        public double Length { get; private set; }
```

Comment:
Длина вектора однозначно определяется его координатами. Она не может изменяться независимо от них.
Её следовало сделать вычисляемым свойством только для чтения. Сеттер нужно убрать совсем.
```suggestion
        public double Length 
        { 
             get
             {
                 return Math.Sqrt(X*X + Y*Y + Z*Z);
             }
        }
```